eng/Versions.props

```diff
@@ -31,7 +31,7 @@
     <CodeStyleLayerCodeAnalysisVersion>3.3.1</CodeStyleLayerCodeAnalysisVersion>
     <MicrosoftCodeAnalysisTestingVersion>1.0.1-beta1.20067.1</MicrosoftCodeAnalysisTestingVersion>
     <CodeStyleAnalyzerVersion>3.5.0-beta3-20078-04</CodeStyleAnalyzerVersion>
-    <VisualStudioEditorPackagesVersion>16.4.248</VisualStudioEditorPackagesVersion>
+    <VisualStudioEditorPackagesVersion>16.5.173-g9c5f5bc040</VisualStudioEditorPackagesVersion>
```

Comment:
Was this needed for new APIs or just so we could have tests that validated the behavior?

Reply:
This is just to make sure tests work correctly. No new API involved.